image-index.md

```diff
@@ -87,7 +87,11 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`features`** *array of strings*
 
-        This property is RESERVED for future versions of the specification.
+        This OPTIONAL property specifies an array of strings, each specifying a mandatory hardware/host feature. Examples are:
+         - CPU type (*haswell*, *broadwell*, *skylake*, *ryzen*)
+         - GPU [NVIDIA Compute Capability](https://developer.nvidia.com/cuda-gpus) (3.7, 6.1)
+         - Host name, to separate different types in a data center without flushing out all the specifics (*ComputeNode-20190213*, *storage-2019*)
+        By using this feature flag, an image index is able to provide specific images for certain hosts.
```

Comment:
In order for interoperability, we need to have either a standard set of well-understood names, or some way parsing well-structured feature names (`cpu-feature:avx512` and `cpu-type:haswell` for instance) in order to be able to handle unknown names.

I am **very** un-nerved by the hostname example, and would like to know how do we handle an image-spec implementation which wants to extract an image but doesn't have that hostname? Should it ignore the field? If so, how is the field mandatory if you can ignore it?